packages/cojson/src/coValueCore/branching.ts

```diff
@@ -40,14 +40,25 @@ export function getBranchHeader({
 export function getBranchId(
   coValue: CoValueCore,
   name: string,
-  ownerId: RawCoID,
+  ownerId?: RawCoID,
 ): RawCoID {
   if (!coValue.verified) {
     throw new Error(
       "CoValueCore: getBranchId called on coValue without verified state",
     );
   }
 
+  if (!ownerId) {
+    const header = coValue.verified.header;
+
+    // Group and account coValues can't have branches, so we return the source id
+    if (header.ruleset.type !== "ownedByGroup") {
+      return coValue.id;
+    }
```

Comment:
Changed the result of branching a group into a noop.

Doing this because people might have groups/accounts in their schemas, and I don't want that to become a blocker.